apps/web/src/components/ShowResult/ShowResult.tsx

```diff
@@ -0,0 +1,289 @@
+          >
+            <svg
+              aria-hidden="true"
+              id="theme-toggle-dark-icon"
+              className="w-5 h-5"
+              fill="currentColor"
+              viewBox="0 0 20 20"
+              xmlns="http://www.w3.org/2000/svg"
+            >
+              <path d="M17.293 13.293A8 8 0 016.707 2.707a8.001 8.001 0 1010.586 10.586z"></path>
+            </svg>
+            <svg
+              aria-hidden="true"
+              id="theme-toggle-light-icon"
+              className="hidden w-5 h-5"
+              fill="currentColor"
+              viewBox="0 0 20 20"
+              xmlns="http://www.w3.org/2000/svg"
+            >
+              <path
+                d="M10 2a1 1 0 011 1v1a1 1 0 11-2 0V3a1 1 0 011-1zm4 8a4 4 0 11-8 0 4 4 0 018 0zm-.464 4.95l.707.707a1 1 0 001.414-1.414l-.707-.707a1 1 0 00-1.414 1.414zm2.12-10.607a1 1 0 010 1.414l-.706.707a1 1 0 11-1.414-1.414l.707-.707a1 1 0 011.414 0zM17 11a1 1 0 100-2h-1a1 1 0 100 2h1zm-7 4a1 1 0 011 1v1a1 1 0 11-2 0v-1a1 1 0 011-1zM5.05 6.464A1 1 0 106.465 5.05l-.708-.707a1 1 0 00-1.414 1.414l.707.707zm1.414 8.486l-.707.707a1 1 0 01-1.414-1.414l.707-.707a1 1 0 011.414 1.414zM4 11a1 1 0 100-2H3a1 1 0 000 2h1z"
+                fill-rule="evenodd"
+                clip-rule="evenodd"
+              ></path>
+            </svg>
+            <span className="sr-only">Toggle dark mode</span>
+          </button>
+          <div
+            id="tooltip-toggle"
+            role="tooltip"
+            className="absolute z-10 inline-block px-3 py-2 text-sm font-medium text-white transition-opacity duration-300 bg-gray-900 rounded-lg shadow-sm tooltip opacity-0 invisible"
+            style={{
+              position: "absolute",
+              inset: "auto auto 0px 0px",
+              margin: "0px",
+              transform: "translate3d(1229.6px, -61.6px, 0px)",
+            }}
+            data-popper-placement="top"
+          >
+            Toggle dark mode
+            <div
+              className="tooltip-arrow"
+              data-popper-arrow=""
+              style={{
+                position: "absolute",
+                left: "0px",
+                transform: "translate3d(68.8px, 0px, 0px)",
+              }}
+            ></div>
+          </div>
+          <a
+            href="/pro/#pricing"
+            className="xl:hidden inline-flex items-center text-white bg-blue-700 hover:bg-blue-800 focus:ring-4 focus:ring-blue-300 font-medium rounded-lg text-sm px-5 py-2.5 text-center ml-3"
+          >
+            Pricing
+          </a>
+          <a
+            href="/pro/#pricing"
+            className="hidden xl:inline-flex items-center text-white bg-blue-700 hover:bg-blue-800 focus:ring-4 focus:ring-blue-300 font-medium rounded-lg text-sm px-5 py-2.5 text-center ml-3"
+          >
+            Pricing &amp; FAQ
+          </a>
+        </div>
+      </div>
```

Comment:
Toàn bộ phần này là của Navbar thì m xóa hết đi hầy, component show Result thì chỉ cần show kết quả là được rồi, còn Navbar thì sẽ được thêm vào trong page với kết quả sau.

Reply:
oki